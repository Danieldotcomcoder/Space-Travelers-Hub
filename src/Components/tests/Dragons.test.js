import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Dragons from '../Dragons';
import store from '../../Redux/configureStore';

it('Renders Dragons Page', () => {
  const dragon = renderer.create(
    <Provider store={store}><Dragons /></Provider>,
  ).toJSON();
  expect(dragon).toMatchSnapshot();
});
