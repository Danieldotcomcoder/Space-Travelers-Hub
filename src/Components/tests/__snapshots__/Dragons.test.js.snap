// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Dragons Page 1`] = `
<div
  className="dragon-c"
/>
`;
